### ISSUE TITLE
MobileViT

### ISSUE THREAD

**Issue by ludmila3** [huggingface/transformers#21221]
### System Info

- `transformers` version: 4.26.0.dev0
- Platform: Linux-5.15.0-1027-gcp-x86_64-with-debian-bullseye-sid
- Python version: 3.7.12
- Huggingface_hub version: 0.11.0
- PyTorch version (GPU?): 1.12.1 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?

 @amyeroberts and @NielsRogge

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1.Run run_image_classification.py with MobileViT models (mobilevit-x-small)

### Expected behavior

Breaks first on Normalize function

**Comment by NielsRogge** [comment]
Answered here: https://github.com/NielsRogge/Transformers-Tutorials/issues/241
